Modules/System/Base64 Convert/src/Base64Convert.Codeunit.al

```diff
@@ -35,6 +58,19 @@ codeunit 4110 "Base64 Convert"
     begin
         exit(Base64ConvertImpl.ToBase64(String, InsertLineBreaks));
     end;
+    /// <summary>
+    /// Converts the value of the input string to its equivalent string representation that is encoded with base-64 digits.
+    /// </summary>
+    /// <param name="String">The string to convert.</param>
+    /// <param name="InsertLineBreaks">Whether insert line breaks in the output or not.
```

Comment:
Whether to insert line breaks in the output.

Reply:
Just a copy'paste of how Microsoft is writing this on existing methods.